### ISSUE TITLE
Missing Test Case - 1502. Can Make Arithmetic Progression From Sequence

### ISSUE THREAD

**Issue by harshitverma1701** [LeetCode-Feedback/LeetCode-Feedback#6823]
#### Your LeetCode username
harshit_1701 

#### Category of the bug
- [ ] Missing Test Cases 

#### Description of the bug
A missing Test Case, I guess would be helpful to coders attempting the question-1502 for next time

#### Code you used for Submit/Run operation

```
    bool canMakeArithmeticProgression(vector<int>& arr) {
        
        int n=arr.size();
        if(n==2)
            return true;
        sort(arr.begin(),arr.end());
        if((arr[1]-arr[0]==arr[2]-arr[1]) and (arr[n-1]-arr[n-2]==arr[n-2]-arr[n-3]))
            return true;
     
        
        return false;
    }
```

#### Language used for code
 C++ 

#### Expected behavior
 Expected behavior would be if the mentioned code had failed to run 
in this test case:'
[1,2,3,7,9,10,11] 

#### Screenshots

IMAGE ON A SIMPLE COMPILER:
![Screenshot (344)](https://user-images.githubusercontent.com/72984439/160225827-ee370f1e-b8d7-4724-8d69-a6fec50a048c.png)

IMAGE ON LEETCODE:
![Screenshot (345)](https://user-images.githubusercontent.com/72984439/160225865-3611a486-9968-46e0-a085-651443b478f0.png)

#### Additional context

Please clear the bug.
My apologies if I was not clear enough.
In a nutshell, 
THIS TC: [1,2,3,7,9,10,11]
SHOULD NOT BE ACCEPTED IN ARITHMETIC PROGRESSION...
BUT ITS ACCEPTING IN MY ABOVE MENTIONED CODE...
 
Thank you........

**Comment by LC-Sue** [comment]
Hi @harshitverma1701 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Yani** [comment]
Thank you for your time. We've used your feedback to update the problem. Your LeetCode account has received 100 LeetCoins as the reward for this feedback. We appreciate your support!
